src/Packetizer/IsoChecksum.c

```diff
@@ -0,0 +1,62 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "IsoChecksum.h"
+
+#define MAKE_UINT16(hi, lo) (uint16_t)(((hi) << 8u) | (lo))
+#define REDUCE(x) (((x)&0xFFu) + ((x) >> 8u))
+
+static uint16_t
+fletcher16(const uint8_t* const data, const size_t bytes)
```

Comment:
This is code that we re-use... so I would consider to leave it as is. But as it is quite old, we might want to bring it up-to-date with our current standards.

Reply:
What do you mean? Just the issues you've noticed below or something else?

Reply:
Naming and magic numbers.... This was done in other comments.